src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1838,6 +1838,18 @@
   <data name="CompatibilityRegisterThisProgramForRestart" xml:space="preserve">
     <value>Register this program for restart</value>
   </data>
+  <data name="Run" xml:space="preserve">
+    <value>Run</value>
+  </data>
```

Comment:
It might be more user friendly to title the option as `Start window`.

Reply:
I agree that `run` is not really user friendly, but file explorer uses exactly the same word. `Launch mode` would also be an alternative, what do you think would be best?

Reply:
Let's go with `Start window`. It's true that File Explore uses this term, but we can do better 🙂